templates/desktop/flavours.html

```diff
@@ -78,62 +86,90 @@ <h3 class="p-heading-icon__title">Edubuntu</h3>
+      <a href="https://kubuntu.org/getkubuntu/" class="p-button--positive">Get Kubuntu</a>
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a href="https://www.kubuntuforums.net/forum">Join the Kubuntu community&nbsp;&rsaquo;</a></li>
+        <li class="p-list__item"><a href="https://twitter.com/kubuntu">Follow Kubuntu&nbsp;&rsaquo;</a></li>
+      </ul>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-start-large-4 col-9">
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/7c6fa2da-kubuntu-2204-screenshot.png",
+        alt="",
+        width="2560",
+        height="1440",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
   </div>
 </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-6">
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
       <div class="p-heading-icon">
         <div class="p-heading-icon__header">
           {{
             image(
               url="https://assets.ubuntu.com/v1/6ac4ba34-lubuntu-logo.svg",
-              alt="Lubuntu icon",
+              alt="",
               height="64",
               width="64",
               hi_def=True,
               loading="lazy",
-              attrs={"class": "p-heading-icon__img"}
+               attrs={"class": "p-heading-icon__img", "style": "margin-top: 0.5rem;"},
             ) | safe
           }}
-          <h3 class="p-heading-icon__title">Lubuntu</h3>
-        </div>
-        <div>
-          <p>Lubuntu is designed to be a simple, easy to use system that is light, fast and modern. Lubuntu provides the LXQt desktop environment which is focused on Qt technologies. Lubuntu comes with the essential applications and services needed to browse the Internet, chat, play and be productive.</p>
-          <p><a href="https://lubuntu.me" class="p-button--positive">Get Lubuntu</a></p>
-          <p><a href="https://discourse.lubuntu.me">Join the Lubuntu community&nbsp;&rsaquo;</a></p>
-          <p><a href="https://twitter.com/lubuntuofficial">Follow Lubuntu&nbsp;&rsaquo;</a></p>
+          <h2 class="p-heading-icon__title">Lubuntu</h2>
         </div>
       </div>
     </div>
-    <div class="col-6 u-vertically-center u-align--center u-hide--small u-hide--medium">
-      {{ image (
+    <div class="col">
+      <p>Lubuntu is designed to be a simple, easy to use system that is light, fast and modern. Lubuntu provides the LXQt desktop environment which is focused on Qt technologies. Lubuntu comes with the essential applications and services needed to browse the Internet, chat, play and be productive.</p>
+      <a href="https://lubuntu.me" class="p-button--positive">Get Lubuntu</a>
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a href="https://discourse.lubuntu.me">Join the Lubuntu community&nbsp;&rsaquo;</a></li>
```

Comment:
At the time of checking this link wasn't working for me

Reply:
@juanruitina could you confirm the link here? (I'd tag the pdm but it looks like he's OOO)

Reply:
Spoke to Juan on MM, it looks like this is a temporary outage on their part so we're going to leave as is for now. This is the correct link